pysqueezebox/player.py

```diff
@@ -964,6 +971,14 @@ async def async_clear_playlist(self, timeout: float = TIMEOUT) -> bool:
             return False
         return await self._wait_for_property("playlist", None, timeout)
 
+    async def async_play_announcement(self, media_id: str) -> bool:
+        """Play announcement url."""
+        if not await self.async_command("playlist", "play", media_id, "Announcement"):
+            return False
+        await self.async_set_repeat("none")
+
+        return await self._wait_for_property("mode", "stop", 120)
+
```

Comment:
I think we should have the pause, save, and resume functionality built in here. I'll move the code over from your HA PR.

Reply:
Yes, as per discord, I think I agree